README.md

```diff
@@ -1,38 +1,36 @@
 # mame2003-libretro
-MAME as it was in 2003, and using the libretro API. Accepts romset **MAME 0.78**. Any file-not-found issues will be because your romset is wrong, or incomplete (e.g. you're trying to run a clone .zip, without the parent .zip present).
+MAME as it was in 2003, using the libretro API. Suitable for lower-end devices that would struggle to run current versions of MAME (later versions of MAME are increasingly accurate, thus can perform worse).
 
-Suitable for lower-end devices that would struggle to run current versions of MAME (later versions of MAME are increasingly accurate, thus can perform worse).
+**Note**: The MAME 2003 core accepts **MAME 0.78** ROMs. Each version of an arcade emulator must be used with ROMs that have the same exact version number. For example, MAME 0.37b5 ROMsets are required by the MAME 2000 emulation, but MAME 0.37b5 sets will not work correctly with the MAME 2003 or MAME 20010 emulator cores, which requires MAME 0.78 and MAME 0.139 ROM sets.
 
```

Comment:
MAME 20010 > MAME 2010.

which requires MAME 0.78 and MAME 0.139 ROM sets
>
which require MAME 0.78 and MAME 0.139 ROM sets, respectively.